recipes/heinz/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.0" %}
```

Comment:
2.0.1?

Reply:
I define the version 0.0.1 as the changes of the dependent library due to the process of compilation, which is irrelevant to the functionality and performance of Heinz. Especially, 2.0.1 stands for CPLEX 12.7; whereas 2.0.0, 12.8.
2.0 is still its main version number of Heinz.

Reply:
@bgruening  To avoid the confusion in future, I have made the change to keep version number consistent today. :smiley:

Reply:
@bgruening  Still, it failed on MacOS with the following error message. Considering Heinz is mainly deployed via Galaxy, I will skip MacOS for now until the compilation issue is solved. 
I have just updated the recipe to skip MacOS and removed everything about it.
```
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [print] Error 1
make[1]: *** [CMakeFiles/print.dir/all] Error 2
make: *** [all] Error 2
BUILD START: heinz-2.0-0
    (actual version deferred until further download or env creation)
```